umbraco/Umbraco.CMS.Backoffice
# Bugfix: Render Property Mandatory marker

## Description

Fixes: https://github.com/umbraco/Umbraco-CMS/issues/16441

This PR renders a mandatory marker based on the property type validation settings.

<img width="1015" alt="Screenshot 2024-07-01 at 11 52 51" src="https://github.com/umbraco/Umbraco.CMS.Backoffice/assets/6078361/58b92e38-9785-4f3d-9927-288db6f4df1a">

## How to test:
* Set some properties to mandatory on a document type. Ensure that these are visually marked on a document with a `*` symbol.

## Types of changes
